docker/scripts/entrypoint.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
```

Comment:
Do you think it's good to put something like `Copied from http://.....` so that we know where this script originally came from? Since it diverges in code style to what we are used to...

Reply:
done